lib/main.dart

```diff
@@ -18,20 +18,29 @@ class MyApp extends StatelessWidget {
       theme: ThemeData(
         primarySwatch: Colors.teal,
         textTheme: TextTheme(
+
           // MenuListItems
           headline1: GoogleFonts.openSans(
             fontSize: 28,
           ),
+
           // ProfileText
           headline2: GoogleFonts.roboto(
             fontSize: 20,
             fontWeight: FontWeight.w500,
           ),
+
           //TextFields
           headline3: GoogleFonts.mukta(
             fontSize: 18,
             fontWeight: FontWeight.w300,
           ),
+
+          // SelectedMenuItem
+          headline4: GoogleFonts.openSans(
+            fontSize: 28,
+            color: Colors.white,
+          ),
```

Comment:
@ahamzatariq , Why do we have all the css in the main.dart file? Should it not be somewhere more specific? maybe `style.dart`?

Reply:
Will do in next PR.

Reply:
Done. In PR#3